src/controllers/accounts/accounts.ts

```diff
@@ -203,10 +210,9 @@ export class AccountsController extends EventEmitter {
     const defaultSelectedAccount = getDefaultSelectedAccount(accounts)
     if (defaultSelectedAccount) {
       await this.#selectAccount(defaultSelectedAccount.addr)
+      await this.updateAccountState(defaultSelectedAccount.addr)
```

Comment:
This is much better compared to updating all account states, I agree. However, what we discussed in https://github.com/AmbireTech/ambire-app/issues/2952 is that fetching the initial account state should not block the account import process.

Can we enhance (speed up) this even further by triggering, but not awaiting the selected account state update here?